edge/controller/model/session_handlers.go

```diff
@@ -151,16 +151,27 @@ func (handler *SessionHandler) HandleList(queryOptions *QueryOptions) (*SessionL
 	return result, nil
 }
 
+func (handler *SessionHandler) HandleListSessionForEdgeRouter(edgeRouterId string) (*SessionListResult, error) {
+	result := &SessionListResult{handler: handler}
+	query := fmt.Sprintf(`anyOf(apiSession.identity.edgeRouterPolicies.edgeRouters) = %v and `+
```

Comment:
mmm the query power!

Reply:
Glad you mentioned it, b/c I noticed the query was wrong in several ways. Had to add ability to check for empty sets in query lang, see netfoundry/ziti-foundation#7